src/doublyLinkedList.js

```diff
@@ -0,0 +1,187 @@
+function Node(data) {
+  this.previous = null
+  this.data = data
+  this.next = null
+}
+
+export default class DoublyLinkedList {
+  constructor() {
+    this.length = 0
+    this.head = null
+  }
```

Comment:
You could have added another member this.tail if you chose. You would have to adjust it with many of the operations also.